libraries/AP_HAL_SITL/GPIO.cpp

```diff
@@ -50,7 +49,7 @@ void GPIO::toggle(uint8_t pin)
 
 /* Alternative interface: */
 AP_HAL::DigitalSource* GPIO::channel(uint16_t n) {
-    return new DigitalSource(0);
+    return new DigitalSource(0);  // TODO : should be n ?
```

Comment:
done. Could you look it the solution is right ? I don't find a way to find where it is used, but it is important as it is define the same way in PX4, VRBRAIN, Empty ...